Tasks/DownloadBuildArtifacts/main.ts

```diff
@@ -163,7 +163,7 @@ async function main(): Promise<void> {
                 }
                 else if (artifact.resource.type.toLowerCase() === "filepath") {
                     let downloader = new engine.ArtifactEngine();
-                    let downloadUrl = artifact.resource.downloadUrl.replace("file:", "");
+                    let downloadUrl = decodeURI(artifact.resource.downloadUrl.replace("file:", ""));
```

Comment:
handle URIError in case Url has invalid encoding characters? otherwise we will hit unhandled exception.

Reply:
that shouldn't be possible since this is output of rest call from build. similar to the way there is no need to catch parseInt for buildId